lib/screens/detail_screen.dart

```diff
@@ -178,12 +178,14 @@ class DetailScreen extends HookConsumerWidget {
                                       Row(
                                         children: [
                                           Text(
-                                            data.premiereDate == null
+                                            (data.premiereDate == null ||
+                                                    data.premiereDate?.year ==
```

Comment:
Only checking for premieredate?.year == null should include the premieredate == null check. I think you should be able to safely remove the first check.

Reply:
Thanks :)
I didn't know if this syntax is valid (comming from Kotlin) but good to know.
I will fix this in a day or two